pkg/webui/console/components/pubsub-form/index.js

```diff
@@ -69,23 +74,33 @@ export default class PubsubForm extends Component {
 
     const initialIsMqtt = update && 'mqtt' in initialPubsubValue
     const initialMqttSecure = initialIsMqtt ? initialPubsubValue.mqtt.use_tls : false
+    const initialUseCredentialsMqtt = initialIsMqtt
+      ? Boolean(initialPubsubValue.mqtt.username && initialPubsubValue.mqtt.password)
```

Comment:
see my review comment

```suggestion
      ? Boolean(initialPubsubValue.mqtt.username || initialPubsubValue.mqtt.password)
```